app/_locales/en/messages.json

```diff
@@ -364,6 +364,9 @@
   "contactsSettingsDescription": {
     "message": "Add, edit, remove, and manage your contacts"
   },
+  "continue": {
+    "message": "Continue"
```

Comment:
Should we rename the `swapPriceDifferenceAcknowledgementNoFiat` message to `continue`, rather than creating a new message? They are identical in content. Even in context, the same thing is being communicated here.

Reply:
Done in 2bcdc1453